R/pkg/R/functions.R

```diff
@@ -34,6 +34,17 @@ NULL
 #' df <- createDataFrame(cbind(model = rownames(mtcars), mtcars))}
 NULL
 
+#' Make date time functions
+#'
+#' @param year The year to represent, from 1 to 9999
+#' @param month The month-of-year to represent, from 1 (January) to 12 (December)
+#' @param day The day-of-month to represent, from 1 to 31
+#'
+#' @name make_datetime_functions
+#' @rdname make_datetime_functions
+#' @family data time functions
+NULL
```

Comment:
we shouldn't do this. there already a part on `@family date` here https://github.com/apache/spark/blob/a57754ccfa74f6c6c365302d454563bd0636a8d5/R/pkg/R/functions.R#L50

Reply:
I am going to add at least one more function `make_timestamp` with the same parameters + `hour`, `min`, `sec` in another PR. Should I add a separate `@family` like:
```
@family make datetime functions
```
?

Reply:
One another option is not-adding to R/Scala/Java parts.